### PR TITLE
Fix: End to end tests do not disable the experiments

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
We were enabling the experiments in a end 2 end test but did not disabled the experiments at the end of the test. That resulted in the experiments staying enabled on the tests executed after.
We also did not have util to disable the experiments. This PR adds an until to disable the experiments. And disables the experiments on the test that enabled them.

## How has this been tested?
I checked that the tests failing on Travis (preview and meta-boxes) are not failing anymore.

### CODE REVIEW

File: packages/e2e-test-utils/src/disable-experimental-features.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * Internal dependencies
+ */
+import { setExperimentalFeaturesState } from './enable-experimental-features';
+
+/**
+ * Disables experimental features from the plugin settings section.
+ *
+ * @param {Array} features Array of {string} selectors of settings to enable. Assumes they can be enabled with one click.
+ */
+export async function disableExperimentalFeatures( features ) {
```
Comment (youknowriad) [inline packages/e2e-test-utils/src/disable-experimental-features.js]: It feels weird to have `setExperimentalFeaturesState` and  `enable` in one file but `disable` in another one. We should probably move them all together and rename the file something like `experimental-features`. I also think this should not be part of ` e2e-test-utils` and should be moved to the `experimental` folder of th `e2e-tests`

File: packages/e2e-tests/specs/experimental/block-transforms.test.js [hunk 2]
```diff
@@ -174,4 +175,14 @@ describe( 'Block transforms', () => {
 			}
 		);
 	} );
+
+	afterAll(
```
Comment (gziolo) [inline packages/e2e-tests/specs/experimental/block-transforms.test.js]: Nit: Can this be colocated with the part which enables the same features?

File: packages/e2e-test-utils/src/enable-experimental-features.js [hunk 3]
```diff
@@ -8,12 +8,7 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import { visitAdminPage } from './visit-admin-page';
 
-/**
- * Enables experimental features from the plugin settings section.
- *
- * @param {Array} features Array of {string} selectors of settings to enable. Assumes they can be enabled with one click.
- */
-export async function enableExperimentalFeatures( features ) {
+export async function setExperimentalFeaturesState( features, enable ) {
```
Comment (gziolo) [inline packages/e2e-test-utils/src/enable-experimental-features.js]: This one could be moved to the shared folder to its own file, although I don't think we are strict about it so maybe it's fine as is.

File: packages/e2e-test-utils/src/disable-experimental-features.js (hunk 1 above)
Comment (gziolo) [reply packages/e2e-test-utils/src/disable-experimental-features.js]: Right, I didn't think about it at all. 100% agree!

Comment (jorgefilipecosta) [reply packages/e2e-test-utils/src/disable-experimental-features.js]: Hi @youknowriad, @gziolo, we are exposing enableExperimentalFeatures in the e2e-test-utils package. This util is just used for tests and not used in production code. In this case, we don't need to keep it in e2e-test-utils, and we can remove it from the package, is that right?

Comment (youknowriad) [reply packages/e2e-test-utils/src/disable-experimental-features.js]: yes, it's a dev package,  breaking changes are fine.

**Comment by jorgefilipecosta** [comment]
Hi @youknowriad, @gziolo your suggestions were applied.

**Comment by ellatrix** [comment]
Is this fixing the failing tests in master?

**Review by youknowriad — APPROVED** [review]
Maybe add a CHANGELOG entry for the e2e-test-utils breaking change

**Comment by jorgefilipecosta** [comment]
> Is this fixing the failing tests in master?

Hi @ellatrix yes it fixes the problems on master and should be merged soon (just waiting for the green light after adding the changelog entry).

**Pull request merged**